### DISCUSSION TITLE
Couldn’t parse iCalendar

### DISCUSSION THREAD

**Discussion by wairds** [bitfireAT/icsx5#71] [Q&A]
I recently received the message ‘ Couldn’t parse iCalendar’ on my Samsung S20.
The error appears to be related to:
‘ Caused by: net.fortuna.ical4j.data.ParserException: Error at line 1:Expected [BEGIN], read [<!DOCTYPE html PUBLIC ]’
Can somebody please how I can identity what caused this error and how I can correct it.
Thank you

**Comment by rfc2822** [comment]
Hi,
Sounds like your calender URL doesn't return an iCalendar (.ics) but a Web page (HTML). Please check your URL (for instance with a browser) - it should return an iCalendar.

**Reply by wairds** [reply]
Thank you for your response. I tried as you suggested and it did return an iCalendar.

**Reply by rfc2822** [reply]
Then it probably checks the client and blocks ICSx5... can you post the URL or is it confidential?

**Reply by wairds** [reply]
It is confidential. However I use the same URL in iOS and there is no issue?

**Reply by rfc2822** [reply]
> It is confidential. However I use the same URL in iOS and there is no issue?

Maybe they don't block iOS. :) But you can send the URL to play@bitfire.at or via https://www.davx5.com/support, it will then be treated confidentially
